core/hw/pvr/ta_ctx.cpp

```diff
@@ -15,6 +15,28 @@ tad_context ta_tad;
 TA_context*  vd_ctx;
 rend_context vd_rc;
 
+#if ANDROID
```

Comment:
Any reason to not merge with this OS_aligned_malloc? I'm afraid of symbols clashing in the future.

Reply:
The NDK keeps relocating the definition between the base and support packages, so that is pulled from the NDK and only required on Android. It's actually just a workaround to force Android to accept that the definition is there (and not error saying it isn't). It may be fixed in the NDK some day, and then broken again, and then fixed. It's kinda what they do.